lib/cc/engine/bundler_audit.rb

```diff
@@ -12,3 +13,16 @@ module BundlerAudit
     end
   end
 end
+
+# Patch Bundler::Audit::Scanner to prevent network access during insecure
+# source checks
+
+module Bundler
+  module Audit
+    class Scanner
```

Comment:
When monkey patching, we've often used [`module_eval`](https://github.com/codeclimate/codeclimate-rubocop/blob/master/lib/rubocop/cop/method_complexity_patch.rb) in the past: it's slightly safer because if the external dependency changes their structure in the future, it'll blow up instead of just silently creating a new class. Though you're accessing the same class elsewhere, so I guess it would blow up anyway.